### PR TITLE
Emit events for read receipts

### PR DESCRIPTION
**Pull request by dbkr**

### CODE REVIEW

File: lib/models/room.js [hunk 1]
```diff
@@ -527,3 +531,14 @@ module.exports = Room;
  *   var newName = room.name;
  * });
  */
+
+/**
+ * Fires whenever a receipt is received for a room
+ * @event module:client~MatrixClient#"Room.receipt"
+ * @param {event} event The receipt event
+ * @param {Room} room The room whose Room.name was updated.
```
Comment (Kegsay) [inline lib/models/room.js]: "The room whose receipts were updated."

**Comment by Kegsay** [comment]
Looks wonderful, but I'd like a test please before LGTMing :) Look at `spec/unit/room.spec.js` for some receipt tests (starting at `:670`). You can just add a `room.on("Room.receipt", ...` in the test itself.

**Comment by dbkr** [comment]
Done!

**Comment by Kegsay** [comment]
LGTM

**Pull request merged**